src/IonTimestamp.ts

```diff
@@ -185,6 +185,27 @@ export class Timestamp {
     );
   }
 
+  /**
+   * Static factory method for creating a Timestamp from a JavaScript Date object
+   *
+   * @param date - valid JavaScript date object
+   */
+  static fromDate(date: Date): Timestamp {
+    const seconds = new Decimal(
+      date.getSeconds(),
+      date.getMilliseconds() / 1000
+    );
+    return new Timestamp(
+      date.getTimezoneOffset() / 60,
```

Comment:
I'm not sure this needs to be divided by 60. Could you double check the units here?